scripts/sipify.pl

```diff
@@ -1189,6 +1222,18 @@ sub detect_non_method_member{
                             }
                         }
                     }
+
+                    if ( $is_qt6 eq 1 and $enum_member ne "" )
+                    {
+                      my $basename = join( ".", @CLASS_AND_STRUCT );
+                      if ( $basename ne "" ){
+                        $enum_member =~ s/^None$/None_/;
```

Comment:
I think this should fatally abort -- it would be good to have a hard block preventing use of None as an enum value, and force the committer to choose a different explanatory name.

Reply:
There are already some : 
QgsMeshDatasetGroup.None
QgsMeshRendererScalarSettings.None
QgsProviderRegistry.None
QgsRasterBandStats.None
QgsRasterMinMaxOrigin.None

SIP/PyQt doesn't complain but there is no way to access then (or failed to find one), so we may consider there are not part of the API, and rename them. No?

Reply:
> , so we may consider there are not part of the API, and rename them

Exactly! 👍